learn/safe-core/safe-core-account-abstraction-sdk/auth-kit.md

```diff
@@ -115,6 +201,72 @@ await signer.signTransaction(tx);
 await signer.signMessage(message);
 ```
 
-## Example
+## Fix Webpack 5 Polyfills Issue
```

Comment:
Hey @ademidun I think would be enough with a link to this known problem to any website helping with this. Is a very long section unrelated to the kit itself 😬

Reply:
Done: f47c33b